extensions/amp-lightbox-gallery/1.0/test/validator-amp-lightbox-gallery.html

```diff
@@ -0,0 +1,109 @@
+
+</head>
+<body>
+  <!-- Valid -->
+  <amp-img src="/awesome.png" width="300" height="300" lightbox></amp-img>
+  <amp-img src="/awesome.png" width="300" height="300" lightbox="a"></amp-img>
+  <amp-img src="/awesome.png" width="300" height="300" lightbox lightbox-thumbnail-id="a"></amp-img>
+
+  <!-- Invalid: missing value for lightbox-thumbnail-id -->
+  <amp-img src="/awesome.png" width="300" height="300" lightbox lightbox-thumbnail-id></amp-img>
+
+  <!-- Valid -->
+  <amp-video
+    lightbox
+    src="../av/ForBiggerJoyrides-short.mp4"
+    width="720"
+    height="405"
+    layout="responsive"
+    controls></amp-video>
+  <amp-youtube
+    lightbox
+    data-videoid="lBTCB7yLs8Y"
+    width="358"
+    height="204"
+    layout="responsive"></amp-youtube>
+
+  <!-- Valid -->
+  <amp-base-carousel width=400 height=300 layout=responsive lightbox></amp-base-carousel>
+  <amp-base-carousel width=400 height=300 layout=responsive lightbox>
+      <amp-img src="/awesome.png" width="300" height="300" lightbox></amp-img>
+      <amp-img src="/awesome.png" width="300" height="300" lightbox-exclude></amp-img>
+  </amp-base-carousel>
+  <amp-base-carousel width=400 height=300 layout=responsive>
+      <amp-img src="/awesome.png" width="300" height="300" lightbox></amp-img>
+      <amp-img src="/awesome.png" width="300" height="300"></amp-img>
+  </amp-base-carousel>
+
+  <amp-base-carousel width=400 height=300 layout=responsive lightbox></amp-base-carousel>
+  <amp-base-carousel width=400 height=300 layout=responsive lightbox>
+      <amp-img src="/awesome.png" width="300" height="300" lightbox></amp-img>
+      <amp-img src="/awesome.png" width="300" height="300" lightbox-exclude></amp-img>
+  </amp-base-carousel>
+  <amp-base-carousel width=400 height=300 layout=responsive>
+      <amp-img src="/awesome.png" width="300" height="300" lightbox></amp-img>
+      <amp-img src="/awesome.png" width="300" height="300"></amp-img>
+  </amp-base-carousel>
+
+ <!-- Valid: carousel with video, img, youtube, ad -->
+  <amp-base-carousel width=400 height=300 layout=responsive lightbox>
+    <amp-video
+      src="../av/ForBiggerJoyrides-short.mp4"
+      width="720"
+      height="405"
+      layout="responsive"
+      controls></amp-video>
+    <amp-youtube
+      lightbox-thumbnail-id="a"
+      data-videoid="lBTCB7yLs8Y"
+      width="358"
+      height="204"
+      layout="responsive"></amp-youtube>
+    <amp-img src="/awesome.png" width="300" height="300" lightbox-exclude></amp-img>
+    <amp-ad width="300"
+      height="250"
+      type="a9"
+      data-aax_size="300x250"
+      data-aax_pubname="test123"
+      data-aax_src="302">
+    </amp-ad>
+  </amp-base-carousel>
+
+  <!-- Invalid: missing value for lightbox-thumbnail-id -->
+  <amp-img src="/awesome.png" width="300" height="300" lightbox lightbox-thumbnail-id></amp-img>
+  <!-- Invalid: invalid use of lightbox-exclude -->
```

Comment:
nit: how is it an invalid use?

Reply:
It looks like `lightbox-exclude` is specific for children of lightboxed carousel elements (as in, lightbox all the elements except the ones with `lightbox-exclude`)

Reply:
the nit was meant to be to include that detail in the comment!

Reply:
oh 😅